app/models/preservica/bitstream.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Bitstream
+  include PreservicaObject
+
+  def initialize(preservica_client, content_id, generation_id, id)
+    @preservica_client = preservica_client
+    @id = id
+    @content_id = content_id
+    @generation_id = generation_id
+  end
+
+  def checksum
```

Comment:
There's the potential for multiple checksum algorithms, which will break this.  Please rename to sha512_checksum and retrieve based on the `<xip:FixityAlgorithmRef>`